web-scraping-bs4/scrape_jobs.py

```diff
@@ -2,28 +2,31 @@
 from bs4 import BeautifulSoup
 
 
-URL = "https://www.monster.com/jobs/search/?q=Software-Developer\
-        &where=Australia"
+URL = "https://realpython.github.io/fake-jobs/"
 page = requests.get(URL)
 
 soup = BeautifulSoup(page.content, "html.parser")
 results = soup.find(id="ResultsContainer")
 
 # Look for Python jobs
-python_jobs = results.find_all("h2", string=lambda t: "python" in t.lower())
-for p_job in python_jobs:
-    link = p_job.find("a")["href"]
-    print(p_job.text.strip())
-    print(f"Apply here: {link}\n")
+print("PYTHON JOBS\n==============================\n")
+python_jobs = results.find_all(
+    "h2", string=lambda text: "python" in text.lower()
+)
+python_job_elems = [h2_elem.parent.parent.parent for h2_elem in python_jobs]
```

Comment:
Let's align variable names with the ones used in the tutorial after you update it :blush: